Makefile

```diff
@@ -34,7 +34,8 @@ TEST_PRODUCTION_BINARY = false
 
 # Build sources
 # TODO: run tasks parallel when in dev mode
-build-source: checks desktop$/config.json build-calypso build-desktop
+build-source: checks desktop-config$/config-base.json build-calypso build-desktop
+	cp desktop-config$/config-base.json desktop$/config.json
```

Comment:
We're not merging any env specific config into the base config here, so we don't need to go through the `desktop$/merged-config-$(CONFIG_ENV).json` target.